package.json

```diff
@@ -7,7 +7,8 @@
   "license": "Apache-2.0",
   "scripts": {
     "clean": "rm -rf node_modules/ && npm cache clean --force",
-    "update-lock": "npm run clean && npm i --no-progress",
+    "update-dependencies": "npm run clean && npm i --no-progress",
+    "check-dependencies": "npm ls || npm run update-dependencies",
```

Comment:
Maybe add `npm ls --depth=0` to reduce verbosity?

Reply:
Does that mean it will check everything? Or just the direct dependencies and not the peer dependencies?
Can't we use `--silent` instead?

```
 npm ls --loglevel silent >/dev/null
```